spatial-tracking-explainer.md

```diff
@@ -323,7 +354,7 @@ Some XR hardware with inside-out tracking has users establish "known spaces" tha
 Additionally, XR hardware with orientation-only tracking may also provide an emulated value for the floor offset of a `floor-level` reference space. On these devices, it is recommended that the User Agent or underlying platform provide a setting for users to customize this value.
 
 ### Reset Event
-The `XRReferenceSpace` type has an event, `onreset`, that is fired when a discontinuity of the reference space's origin occurs.  This discontinuity may be caused for different reasons for each type, but the result is essentially the same, the perception of the user's location will have changed.  In response, pages may wish to reposition virtual elements in the scene or clear any additional transforms, such as teleportation transforms, that may no longer be needed.  The `onreset` event will fire prior to any poses being delivered with the new origin/direction, and all poses queried following the event must be relative to the reset origin/direction. 
+The `XRReferenceSpace` type has an event, `onreset`, that is fired when there is a discontinuity of the location of the reference space's origin in the physical world.  This discontinuity may be caused for different reasons for each type, but the result is essentially the same, the perception of the user's location will have changed.  In response, pages may wish to reposition virtual elements in the scene or clear any additional transforms, such as teleportation transforms, that may no longer be needed.  The `onreset` event will fire prior to any poses being delivered with the new origin/direction, and all poses queried following the event must be relative to the reset origin/direction. The `transform` value in the `onreset` event data indicates the size of the jump of the reference space origin in the physical world, if known.
```

Comment:
Are you explicitly trying to say that an orientation discontinuity will not cause it?

Reply:
No, I was just referring more explicitly to the "location" "in the physical world", to make clearer what is meant by a discontinuity of the origin.  In general, we've been using "location" when we want to more generically refer to the position/orientation of something.  Some readers may assume that "location" just means "position", but even then, 99% of the time, a reference space discontinuity will involve a discontinuity of "position", and so that is still the right mental model.